### DISCUSSION TITLE
Resources

### DISCUSSION THREAD

**Discussion by erlend-sh** [fishfight/punchy#150] [Planning]
# Assets

## Sound

**UI (temporary):**
https://www.kenney.nl/assets/interface-sounds
https://www.kenney.nl/assets/ui-audio

# Tutorials

wip

**Comment by erlend-sh** [comment]
☝️ I believe this topic is editable by all managers/owners of this repo? I’d prefer to out this in a Discussion rather than a Wiki, since no additional tab context needs to be added, plus here we can be properly pinged about bigger updates or questions about the content.

**Reply by 64kramsystem** [reply]
I confirm I can edit.

**Comment by zicklag** [comment]
Moved resources to the wiki: https://github.com/fishfolks/punchy/wiki/Assets.

We'll try to see if the wiki works well for our purposes.

**Comment by 64kramsystem** [comment]
I've added the enemies state machine [diagram](https://github.com/fishfolks/punchy/wiki/Architecture#enemy-state-machine). It's very simple, but it should considerably help beginners to grasp the workflow.

Diagrams are written directly in Markdown, since Github supports Mermaid.

**Reply by zicklag** [reply]
Oh, I didn't know GitHub supported mermaid. That's really nice. :+1:

**Comment by zicklag** [comment]
Pushed an update to the [Architecture Page](https://github.com/fishfolks/punchy/wiki/Architecture) with an overview of the modules in the repo after merging #203.

We'll want to try to keep it up to date as we make changes now that it's somewhat fleshed out.

**Reply by erlend-sh** [reply]
Excellent! Lots of good info already.

Could you please start a new discussion topic that’s two-way linked to the architecture page? I’d like for devs to have a dedicated place to discuss these pages, that way we get the best of both worlds, whilst also fulfilling what this topic originally set out to do.

See the bottom of your Fighter State Machine page to see what I mean by two-way linked.
